pkg/utils/gqlclient.go

```diff
@@ -69,6 +69,11 @@ func (request *Request) SetToken(token string) {
 	request.Header.Set("Authorization", token)
 }
 
+// SetUserId sets the Authorization header for the request with the given user id.
+func (request *Request) SetUserId(token string) {
+	request.Header.Set("user_id", token)
```

Comment:
Please, rename parameter:
```suggestion
func (request *Request) SetUserId(userId string) {
	request.Header.Set("user_id", userId)
```